server/src/main/java/com/google/fhir/gateway/ExceptionUtil.java

```diff
@@ -53,7 +53,7 @@ static void throwRuntimeExceptionAndLog(Logger logger, String errorMessage) {
     throwRuntimeExceptionAndLog(logger, errorMessage, null, RuntimeException.class);
   }
 
-  static void throwRuntimeExceptionAndLog(Logger logger, String errorMessage, Exception e) {
+  public static void throwRuntimeExceptionAndLog(Logger logger, String errorMessage, Exception e) {
```

Comment:
Is this needed to be called from outside this package? If not, please remove `public`.

Reply:
Yes, we are using it in a custom access checker helper classes defined in another package.

Reply:
Okay; thanks for the clarification. The util functions like this are not designed to be part of the public interface of this library, but I guess it is probably okay to use them (I am not sure if their interface is great); resolving this for now.